app-modules/portal/src/DataTransferObjects/KnowledgeBaseCategoryData.php

```diff
@@ -45,5 +46,9 @@ public function __construct(
         public string $name,
         public ?string $description,
         public ?string $icon,
+        /** @var null|KnowledgeBaseCategoryData */
```

Comment:
This annotation shouldn't be needed. Just the one below for the collection so that you can define what it is a collection of. But this definition is automatically inferred from the type hinting.